### DISCUSSION TITLE
Double closing brackets happen often

### DISCUSSION THREAD

**Discussion by gijsbeijer** [community/community#13400] [Copilot Conversations]
Hi,

The inline suggestions often include the closing brackets, also when they are already there. 

Using an IDE, mostly when you type the opening bracket, the closing bracket also gets created. Combined with the inline suggestion closing brackets, I often have to remove one of the two.

Would be great if it is possible if there was detection for this.

**Comment by joliss** [comment]
This is probably related: https://github.com/community/community/discussions/15550

**Comment by ihnatenkoo** [comment]
Hello, gijsbeijer. Look for these settings in your IDE
![2022-09-22 20 42 25](https://user-images.githubusercontent.com/89586345/191815788-3ebedb48-6ac4-482c-a335-878cedf7432a.png)
